src/backend/cuda/include/yaksuri_cudai.h

```diff
@@ -23,6 +23,8 @@ extern "C" {
 
 #include <yaksuri_cudai_base.h>
 
+#define YAKSRI_KERNEL_NULL  NULL
```

Comment:
What's this `YAKSRI` namespace?  Did you mean `YAKSURI`?

Reply:
oops, my bad. typo

Reply:
The commit message and commit description still have the same typo.

Reply:
The PR description too.